pkg/peer/webrtc.go

```diff
@@ -133,6 +133,14 @@ func (p *Peer[ID]) onDataChannelReady(dc *webrtc.DataChannel) {
 	dc.OnOpen(func() {
 		p.logger.Info("Data channel opened")
 		p.sink.Send(DataChannelAvailable{})
+
+		startKeepAlive(
+			p.pingInterval,
+			p.keepAliveDeadline,
+			p.pong,
+			p.sendPing,
+			func() { p.sink.Send(LeftTheCall{event.CallHangupKeepAliveTimeout}) },
+		)
```

Comment:
Note, we probably want to try to start this go-routine earlier (once the peer is constructed), so that the peer is thrown away if no data channels are available or if the opening of the data channel takes too long.